README.md

```diff
@@ -107,6 +108,12 @@ You can do `./download-paks --help` for more options.
 ./download-paks build/pkg
 ```
 
+The above snippet stores the downloaded paks in the
+[libpath](https://wiki.unvanquished.net/wiki/Libpath) of the Daemon engine that you are building,
```

Comment:
The “in the libpath” wording may suggest the dpk archives are not stored in `libpath/pkg` but in `libpath`.

Reply:
OK I changed the wording.